test/widgets/action_sheet_test.dart

```diff
@@ -160,9 +160,13 @@ void main() {
         ]);
       }
 
-      await tester.pumpWidget(TestZulipApp(accountId: eg.selfAccount.id,
-        child: const HomePage()));
+      // ZulipApp instead of TestZulipApp so that we can push a route to the nav,
+      // exercising HomePage.buildRoute which (via AccountPageRouteMixin)
+      // makes a PageRoot, which the action sheet needs.
+      await tester.pumpWidget(ZulipApp());
```

Comment:
Hmm, do we push a route to the nav in these tests? I don't see where.

Reply:
Ah indeed; yeah it turned out we don't need to push a route in order to show the inbox, since it's on HomePage, which automatically appears. We still need `ZulipApp` instead of `TestZulipApp`, though, to get the `PageRoot` included, since that happens via `HomePage.buildRoute`.

Reply:
I think it'd make sense to have TestZulipApp provide a PageRoot too. It's setting `home`, which is basically a simplified version of the job that `onGenerateInitialRoutes` does.

Reply:
Sure, seems reasonable.